metadata-ingestion/build.gradle

```diff
@@ -171,6 +171,9 @@ task testSlowIntegration(type: Exec, dependsOn: [installDevTest]) {
 task docGen(type: Exec, dependsOn: [codegen, installDevTest]) {
   commandLine 'bash', '-c', "source ${venv_name}/bin/activate && ./scripts/docgen.sh"
 }
+task buildWheel(type: Exec, dependsOn: [install, codegen]) {
+  commandLine 'bash', '-c', "source ${venv_name}/bin/activate && " + 'pip install build && RELEASE_VERSION="\${RELEASE_VERSION:-0.0.0.dev1}" RELEASE_SKIP_TEST=1 RELEASE_SKIP_UPLOAD=1 ./scripts/release.sh'
```

Comment:
What do 0.0.0.dev0 and 0.0.0.dev1 represent? I see we have:
```
def is_dev_mode() -> bool:
    return __version__.endswith("dev0")
```
so does that mean 0.0.0.dev1 is not dev mode?

Reply:
Yep, because it's not installed in "editable" mode.